UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterTile.cs

```diff
@@ -119,28 +121,51 @@ public NetworkInstanceId ParentNetId
 	// is invoked.
 	[SyncVar(hook = nameof(SetParent))] private NetworkInstanceId parentNetId;
 
-	public Vector3Int WorldPosition => MatrixManager.Instance.LocalToWorldInt(position, Matrix);
+	public Vector3Int WorldPositionS => MatrixManager.Instance.LocalToWorldInt(serverPosition, Matrix);
```

Comment:
```suggestion
	public Vector3Int WorldPositionServer => MatrixManager.Instance.LocalToWorldInt(serverPosition, Matrix);
```

I think it would be clearer if you renamed some of these.